src/app/components/Splash/index.tsx

```diff
@@ -4,23 +4,18 @@ import { browser } from 'webextension-polyfill-ts';
 import './style.less';
 
 interface Props {
-  handleContinue(): void;
+  onComplete(): void
 }
 
 export default class Splash extends React.Component<Props> {
   componentDidMount() {
-    if (process.env.APP_CONTAINER === 'page') {
-      browser.storage.local.get('skipSplash').then(value => {
-        if (value && value.skipSplash) {
-          browser.storage.local.remove('skipSplash').then(() => {
-            this.handleContinue();
-          });
-        }
-      });
+    if (process.env.APP_CONTAINER !== 'page') {
+      browser.runtime.openOptionsPage().then(window.close);
```

Comment:
While I don't like adding unnecessary steps for users, I don't like that clicking the icon initially causes the options page to open. I want users to know that clicking the icon will open a popup and not interrupt their current session (normally) and that onboarding is a special case. Reverting this should also fix the issue mentioned in my previous comment above.